internal/distributed/indexcoord/service.go

```diff
@@ -109,6 +105,11 @@ func (s *Server) start() error {
 		return err
 	}
 	log.Debug("indexCoord started")
+	if err := s.indexcoord.Register(); err != nil {
+		log.Error("IndexCoord", zap.Any("register session error", err))
```

Comment:
Error message should be like "Index Register failed", with zap.Error(err)

Reply:
Same as above.